easybuild/easyconfigs/c/Control-FREEC/Control-FREEC-11.5-GCC-8.3.0.eb

```diff
@@ -0,0 +1,59 @@
+# #
+# This is a contribution from HPCNow! (http://hpcnow.com)
+# Copyright::   HPCNow!
+# Authors::     Jordi Blasco <jordi.blasco@hpcnow.com>
+# License::     GPL-v3.0
+# #
+
+easyblock = 'MakeCp'
+easyblock = 'ConfigureMake'
+
+name = 'Control-FREEC'
+version = '11.5'
+
+homepage = 'https://github.com/BoevaLab/FREEC'
+description = """Control-FREEC is a tool for detection of copy-number changes and allelic imbalances (including LOH)
+using deep-sequencing data originally developed by the Bioinformatics Laboratory of Institut Curie (Paris). Since 2016,
+the project has moved to Insitut Cochin, INSERM U1016 (Paris). Control-FREEC automatically computes, normalizes,
+segments copy number and beta allele frequency (BAF) profiles, then calls copy number alterations and LOH."""
+
+
+toolchain = {'name': 'GCC', 'version': '8.3.0'}
+
+builddependencies = [
+    ('Autoconf', '2.69', '', ('GCCcore', '8.3.0')),
+    ('Automake', '1.16.1', '', ('GCCcore', '8.3.0')),
+    ('Autotools', '20180311', '', ('GCCcore', '8.3.0')),
+]
+
+dependencies = [
+    ('R', '3.6.2', '', ('foss', '2019b')),
```

Comment:
This breaks HMNS. This config simply needs to be on the foss level.

Reply:
done!